[quant][graphmode] Rename graph mode quantization API from `*_script` to `*_jit`

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#40212 [quant][graphmode] Rename graph mode quantization API from `*_script` to `*_jit`**

Summary:
based on discussions in https://github.com/pytorch/pytorch/pull/40198

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D22144745](https://our.internmc.facebook.com/intern/diff/D22144745)

jerryzh168 committed:
[quant][graphmode] Rename graph mode quantization API to `quantize_jit`

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Rename graph mode quantization API to `quantize_jit`"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Rename graph mode quantization API to `quantize_jit`"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Rename graph mode quantization API from `*_script` to `*_jit`"


Summary:
based on discussions in https://github.com/pytorch/pytorch/pull/40198

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D22144745](https://our.internmc.facebook.com/intern/diff/D22144745)

[ghstack-poisoned]

z-a-f commented on test/quantization/test_quantize_jit.py:
```diff
@@ -22,17 +22,19 @@
-from torch.quantization.quantize_script import fuse_conv_bn_script
+# torch.quantization.quantize_jit
+from torch.quantization.quantize_jit import (
+    script_qconfig,
+    prepare_jit,
+    convert_jit,
+    prepare_dynamic_jit,
+    convert_dynamic_jit,
+    fuse_conv_bn_jit,
```
nit: Can you put them in alphabetical order -- easier to find. Also, in my personal opinion having separate lines for import (instead of `()`) looks better :)

jerryzh168 replied:
Sure, will put them in alphabetical order, `()` has less things to type I think?

z-a-f reviewed (approved):
LGTM

jerryzh168 committed:
Update on "[quant][graphmode] Rename graph mode quantization API from `*_script` to `*_jit`"


Summary:
based on discussions in https://github.com/pytorch/pytorch/pull/40198

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D22144745](https://our.internmc.facebook.com/intern/diff/D22144745)

[ghstack-poisoned]
